extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/converter/type/QuantityTypeConverter.java

```diff
@@ -140,8 +136,25 @@ protected boolean fromBindingValidation(HmDatapoint dp) {
                 return new QuantityType<Energy>(number, SmartHomeUnits.WATT_HOUR);
             case "m3":
                 return new QuantityType<Volume>(number, SIUnits.CUBIC_METRE);
+            case "100%":
+                // "100%" is a commonly used "unit" in datapoints. Generated channel-type
+                // is of DecimalType, but clients may define a QuantityType if preferred
+                return new QuantityType<Dimensionless>(number.doubleValue() * 100.0, SmartHomeUnits.PERCENT);
+            case "s":
+            case "min":
+            case "minutes":
+            case "day":
+            case "month":
+            case "year":
+            case "":
+            default:
+                return new QuantityType<Dimensionless>(number, SmartHomeUnits.ONE);
         }
-        return new QuantityType<Dimensionless>(number, SmartHomeUnits.ONE);
+    }
+
+    @Override
+    protected void logTraceOrDebug(String format, Object... arguments) {
```

Comment:
It took me a while to get the intention of this method. Here's my suggestion to make it easier understandable.

I would suggest to create another method `getDefaultLogLevelForTypeConverter()`, which returns "trace" in the AbstractTypeConverter and is overridden in this class, where it returns "debug" (you could of course also do this with a protected field instead of a method). Then, change `logTraceOrDebug()` to `logAtDefaultLevel()`, which logs the message at the level specified by `getDefaultLogLevelForTypeConverter()`.
This makes it more explicit that the log level depends on the concrete sub-class. Otherwise, when seing the `logTraceOrDebug()` method in the AbstractTypeConverter, which just logs at trace level, it is hard to understand its name.

Reply:
My idea was to increase logging verbosity for the `QuantityTypeConverter`. The approach that you suggest is easier to understand, so I'll use that